python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -543,12 +541,54 @@ def update_stop_sequence(db_manager: DatabaseManager) -> None:
                 == StaticTrips.static_version_key,
             ),
         )
+        .where(
+            StaticRoutePatterns.static_version_key
+            == sa.func.any(
+                sa.func.array(
+                    sa.select(TempEventCompare.static_version_key)
+                    .distinct()
+                    .scalar_subquery()
+                )
+            ),
+            sa.or_(
+                StaticRoutePatterns.route_pattern_typicality == 1,
+                StaticRoutePatterns.route_pattern_typicality == 5,
+            ),
+        )
+        .order_by(
+            sa.func.coalesce(
+                StaticTrips.branch_route_id, StaticTrips.trunk_route_id
+            ),
+            StaticRoutePatterns.direction_id,
+            StaticRoutePatterns.static_version_key,
+            StaticRoutePatterns.route_pattern_typicality.desc(),
```

Comment:
i think a comment here that notes we want route pattern typicality of 5 first, falling back to 1 where we can't find it.

i think this distinction gets lost pretty easily.